speakers.js

```diff
@@ -0,0 +1,55 @@
+const data = [
+  {
+    spimg: 'src="./assest/speakers/speaker1.png" alt=""',
+    Name: 'yochai benkler',
+    title: '3D Across ArcGIS',
+    dsc: '3D is an essential component of many organizations spatial systems of record and engagement. Esri provides 3D capabilities including customizable and tailored solutions for a wide variety of partners and users',
+  },
+  {
+    spimg: 'src="./assest/speakers/speaker2.png" alt=""',
+    Name: 'kilnam chon',
+    title: 'How to Make Your Web App World-Ready!',
+    dsc: 'In this session we will give an overview of what you need to consider when globalizing your web app and show you how Esri’s APIs, SDKs, and component libraries can help you with that task.',
+  },
+  {
+    spimg: 'src="./assest/speakers/speaker3.png" alt=""',
```

Comment:
- [ ] Great job generating the "Featured speakers" dynamically with JavaScript. The section. :partying_face::+1:💪🏻 